releases/release-1.24/README.md

```diff
@@ -34,60 +34,61 @@ description: |
-| **Begin [Enhancements Freeze]** ( **Time TBC**) | Enhancements Lead | **TBC** | week 3 | [master-blocking], [master-informing] |
-| 1.24.0-alpha.2 released | Branch Manager | **TBC** | Week 4 | |
-| 1.24.0-alpha.3 released | Branch Manager | **TBC** | Week 6 | |
-| **Begin [Burndown]** (Monday, Wednesday, and Friday meetings) | Lead | **TBC** | week 11 | [1.24-blocking], [master-blocking], [master-informing] |
-| **Call for [Exceptions][Exception]** | Lead | **TBC** | week 11 | |
-| Brace Yourself, Code Freeze is Coming | Comms / Bug Triage | **TBC** | week 11 | |
-| **Begin Feature blog freeze** (**Time TBC**) | Comms Lead | **TBC** | week 11 | |
-| 1.24.0-alpha.4 released | Branch Manager | **TBC** | Week 11 | |
-| Release retrospective part 1 (**Time TBC**) | Community | **TBC** | week 11 | |
-| Burndown Meetings daily| Lead | **TBC** | week 13 | |
-| **Begin [Code Freeze]** (**Time TBC**) | Branch Manager | **TBC** | week 13 | |
-| 1.24.0-beta.0 released | Branch Manager | **TBC** | week 13 | |
-| Docs deadline - Open placeholder PRs | Docs Lead | **TBC** | week 13 | |
-| **[Test Freeze]** (**Time TBC**) | Branch Manager | **TBC** | week 14 | |
-| Docs deadline - PRs ready for review | Docs Lead | **TBC** | week 14 | |
-| 1.24.0-rc.0 released | Branch Manager | **TBC** | week 14 | |
-| release-1.24 branch created | Branch Manager | **TBC** | week 14 | |
-| release-1.24 jobs created | Branch Manager | **TBC** | week 14 | |
-| Start final draft of Release Notes | Release Notes Lead | **TBC** | week 14 | |
-| Release blog ready to review (**Time TBC**) | Comms / Docs | **TBC** | week 14 | |
-| Docs complete - All PRs reviewed and ready to merge | Docs Lead | **TBC** | week 15 | |
-| Feature blogs ready to review (**Time TBC**)| Enhancement Owner / SIG Leads | **TBC** | week 15 | |
-| Release Notes complete - reviewed & merged to `k/sig-release` (**Time TBC**) | Release Notes Lead | **TBC** | week 15 | |
-| 1.24.0-rc.1 released | Branch Manager | **TBC** | week 15 | |
-| **v1.24.0 released** | Branch Manager | **TBC** | week 16 | |
-| Release blog published | Comms | **TBC** | week 16 | |
-| **[Thaw]** | Branch Manager | **TBC** | week 16 | |
-| Release retrospective part 2 (**TBC**) | Community | **TBC** | week 17 | |
-| Release retrospective part 3 (**TBC**) | Community | **TBC** | week 17 | |
+| Start of Release Cycle | Lead | Monday 10th January 2022 | week 1 | [master-blocking] |
+| Start Enhancements Tracking | Enhancements Lead | Monday 10th January 2022 | week 1 | |
+| Schedule finalized | Lead | Friday 14th January 2022 | week 1 | |
+| Team finalized | Lead | Friday 14th January 2022 | week 1 | |
+| Production Readiness Soft Freeze | Enhancements Lead | Thursday 20th January 2022 | week 2 | |
+| Start Release Notes Draft | Release Notes Lead | Tuesday 25th January 2022 | week 3 | |
+| **Begin [Enhancements Freeze]** ( **Time TBC**) | Enhancements Lead | Thursday 27th January 2022 | week 3 | [master-blocking], [master-informing] |
```

Comment:
Firm time for enhancements freeze will be 18:00:00 PT

Reply:
:+1: I've noted that down as "02:00 UTC Friday 28th January 2020 / 18:00 PST Thursday 27th January 2022"